tensorflow/core/protobuf/config.proto

```diff
@@ -99,6 +99,22 @@ message GPUOptions {
   // the overall host system performance.
   bool force_gpu_compatible = 8;
 
+  // Whether to merge streams in one stream group. Four types of streams will be
```

Comment:
"Whether to merge data transfer streams into the compute stream in a stream group" might be clearer.